src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -43,7 +52,7 @@ tests:
               - name: "direction"
                 value: 0
               - name: "transitionTime"
-                value: 100
+                value: 300
```

Comment:
Why these 100 to 300 changes?

Reply:
As per latest testplan the Transition time need to be 300

Reply:
I understand this is what is written in the test plan. But when testing the feature in an automated test how does it really validate the feature differently. I see a lot of the tests are now 300. This means 30s to complete the transition. This should also reflect in the wait time before validating the read value. This will increase CI run time while I don't see how it changes the validation of the feature.

Reply:
An easy way to solve that will be to duplicate the step. One of them will contain the value used for the test plan while the other will contain a value that is good enough for CI.
Then the steps can be enabled or not with the use of PICS code. 
by default all PICS code on CI default to true, so for example a new PICS code: PICS_CI can be added and the test harness team will just turn it off.